include/umf/base.h

```diff
@@ -28,7 +28,7 @@ extern "C" {
 #define UMF_MINOR_VERSION(_ver) (_ver & 0x0000ffff)
 
 /// @brief Current version of the UMF headers
-#define UMF_VERSION_CURRENT UMF_MAKE_VERSION(0, 11)
+#define UMF_VERSION_CURRENT UMF_MAKE_VERSION(0, 12)
```

Comment:
soo.. now as we established we'll be doing the 1.0.0 version, is this about to be changed...?

Reply:
For now we have 0.12 tag - imho this variable should be updated to 1.0 with a release